packages/core/src/redux/slices/signalConnection/index.ts

```diff
@@ -29,6 +29,7 @@ import {
 import { Credentials } from "../../../api";
 import { selectAppIsActive } from "../app";
 import { signalEvents } from "./actions";
+export { signalEvents } from "./actions";
```

Comment:
Wouldn't it look better if this is at the bottom of the file? And as this is already imported, maybe the `from "./actions"` is not necessary.

Reply:
Hmmm TS doesn't like any other way I've found to export it, besides:
```typescript
```

Reply:
happy to move it to the bottom if you prefer though, personally I like keeping the two together